src/components/breadcrumb/breadcrumb-step/breadcrumb-step.js

```diff
@@ -2,10 +2,15 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
-import SvgIcon from '@material-ui/core/SvgIcon';
 import Grid from '@material-ui/core/Grid';
 import Fab from '@material-ui/core/Fab';
 import styles from '../breadcrumb.module.css';
+import QuestionMarkIcon from '../../icons/question-mark';
+import CheckmarkIcon from '../../icons/checkmark';
+import DollarSignIcon from '../../icons/dollar-sign';
+import NotesIcon from '../../icons/notes';
+import Notes2Icon from '../../icons/notes-2';
```

Comment:
What is the `Notes2Icon`? Is there a name we can give it that differentiates it from the other notes icon better than with a number?

Reply:
So notes icon is a single note and notes2 is like two notes  on top of each other. I could name them 

notes-single
notes-stacked

Does that work?